src/quote/indexQuoteProvider.ts

```diff
@@ -267,6 +235,7 @@ function getContractType(token: string): FlashMintContractType | null {
 }
 
 function requiresZeroExV1(contractType: FlashMintContractType): boolean {
+  if (contractType === FlashMintContractType.erc4626) return true
```

Comment:
The erc4626  provider now requires zeroExV1 due to dynamic fetching of swap data